doc/source/conf.py

```diff
@@ -268,7 +268,27 @@
 
 htmlhelp_basename = 'scipy'
 
-mathjax_path = "scipy-mathjax/MathJax.js?config=scipy-mathjax"
+
+# Temporary workaround for https://github.com/executablebooks/MyST-NB/issues/590
+# Check if we have access to the internet. If so, use MathJax via its CDN. If not
+# we are most likely building locally without internet access and therefore we use
+# the local vendored MathJax distribution present in the scipy-mathjax directory.
+
+import socket  # noqa: E402
+
+INTERNET_AVAILABLE = False  # assume no internet connection + use local MathJax
+
+try:
+    # Attributed to https://stackoverflow.com/a/33117579/14001839
```

Comment:
This is a bit awkward because SciPy's license is [not compatible with StackOverflow's](https://scipy.github.io/devdocs/dev/core-dev/index.html#licensing).

Reply:
Yes 👍 we cannot use this code and need to come up with our own solution or use a MIT/BSD existing code.

Reply:
Thank you for the comment, I had not noticed that – let me try to find another code snippet that is compatible with SciPy's license and revise this functionality with that.